### DISCUSSION TITLE
当应用被关闭时，正在下载的文件会被删除，这是正常的吗？

### DISCUSSION THREAD

**Discussion by Nio1023** [alex8088/electron-vite#285] [Q&A]
大家好，
我希望通过createInterruptedDownload的API来实现downloadItem的断点续传。
但是，我发现，应用关闭时未下载完的文件会被删除，导致重启客户端后，无法从断点继续下载。
请问，这是electron的规则吗？还是有什么办法可以保留尚未下载完的文件？

**Comment by alex8088** [comment]
建议：可以分块保存，并记录保存位置，程序退出时不致于全部丢失，这样处理也可以减少内存占用
